sample/README.md

```diff
@@ -1,6 +1,64 @@
 <h1>Hazelcast Go Client</h1>
 
+
+In order to run any example, you should have a running Hazelcast instance, if you don't already have one:
+- [download the latest hazelcast release](https://hazelcast.org/download/).
+- [start an instance](http://docs.hazelcast.org/docs/latest/manual/html-single/index.html#using-the-scripts-in-the-package).
+
+When you have a running Hazelcast instance you will see its address and port in console. You should add this to the **client network config** 
+so that the client can connect to the server. 
+
+```
+config := hazelcast.NewConfig()
+config.NetworkConfig().AddAddress("192.168.1.9:5701") // change this with your address
+
+```
+
+**Don't forget to change the address with your servers address.**
```

Comment:
**The address given above is an example. Don't forget to change it to to your server's address.**